front-end/src/pages/AuthorPage.tsx

```diff
@@ -50,13 +51,33 @@ export default function AuthorPage(props: any) {
   const [responseMessage, setResponseMessage] = useState(100);
```

Comment:
On line 47 we should bind the props of this to type `Props`. This is a bit of scope creep. However, since we're changing around the props in this component, I think it would be good to set the prop type to be confident that we're not introducing any bugs

Reply:
agreed, I think the reason why some places we have `props: any` is because like idk how to explain it but there are issues that come up with typing some of the props that get passed in

Reply:
I think the `FollowListPage` component has the typing properly implemented. I want to be a bit firmer with this alone since if we can type the props we can instantly find any `undefined` bugs if they exist

Reply:
If it doesn't work out maybe make an issue and tag me in it so I can take a look. I don't want to block you too long on this considering the timeline

Reply:
I'll make an issue, I know that the problem is that I think when we were working on the AuthorPage, at the time we didn't know how to type `props.location`. That is causing issues

Reply:
We can type `.location` by doing what we have in `FollowListPage`, which should extend the React Router Props interface type

Reply:
I'll take a quick look to see if I can type it in your branch. Feel free to merge whenever though